packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,145 @@
+# Checkout
+Checkout is a process of providing information about shipping and billing details, and picking certain shipping and payment method. Then with all the data, we are able to place an order and pay for it.
+
+## Shipping
+To access shipping details, we can use property of `useShipping` called `shipping`.
+```js{8,16}
+import { useShipping } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      shipping
+    } = useShipping();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      shipping
+    };
+  }
+}
+```
+`shipping` property will return `null` if the address is not saved on the server or just has not been loaded. In order to save shipping details, you could use `save` method:
+```vue{2,15,24}
+<template>
+  <form @submit.prevent="save({ shippingDetails: shippingForm })">
+    <!-- form fields -->
+    <button type="submit" :disabled="loading">Submit</button>
+  </form>
+</template>
+
+<script>
+import { useShipping } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      save,
+      loading,
+      shipping
+    } = useShipping();
+
+    const shippingForm = ref(/* object for shipping details */);
+
+    return {
+      shippingForm,
+      save
+    };
+  }
+}
+</script>
+```
+
+## Shipping providers
```

Comment:
The below paragraph will probably be unable to understand by anyone who doesn't already know what shipping provider component is. The explanation is missing essential information like why we need this component, what is a shipping provider component, what are responsibilities of such component etc.

Reply:
also what is a proper place, please avoid such constructions in the docs - be specific